node/JobRoutes.js

```diff
@@ -0,0 +1,7 @@
+app.get('/JobsTable', async function (req, res) { 
```

Comment:
Ideally, the function would be imported from another module. This facilitates testing and separation of concerns.